Apigee/ManagementAPI/AbstractApp.php

```diff
@@ -900,7 +900,7 @@ protected static function loadCredentials(AbstractApp &$obj, array $credentials)
             }
 
             // Some apps may be misconfigured and need to be populated with their apiproducts based on credential.
-            if (count($obj->apiProducts) == 0) {
+            if (empty($obj->apiProducts)) {
```

Comment:
Is there a reason we need to change this for 7.2?  Are these vars not arrays?

Reply:
They probably are arrays, but the general idea here was to use empty() instead of comparing count to 0. This was the strategy employed for drupal 7 core as well, see https://www.drupal.org/project/drupal/issues/2885610